cookie-banner-rules-list.json

```diff
@@ -1167,7 +1167,7 @@
         "optOut": [
           {
             "name": "FCCDCF",
-            "value": "%5Bnull%2Cnull%2Cnull%2C%5B%22CPgbQkAPgbQkAEsABCBGCjCgAAAAAAAAAAIwAAAOhQD2F2K2kKEkfjSUWYAQBCujIEIBUAAAAECBIAAAAUgQAgFIIAgAAlACAAAAABAQAQCAgAQABAAAoACgACAAAAAAAAAAAAQQAABAAIAAAAAAAAEAQAAAAAQAAAAAAABEhCAAQQAEAAAAAAAAAAAAAAAAAAABAAA.YAAAAAAAAAA%22%2C%221~%22%2C%22FA11CE3A-809E-4B87-BFDE-13E9DE22F419%22%5D%2Cnull%2Cnull%2C%5B%5D%5D"
+            "value": "%5Bnull%2Cnull%2Cnull%2C%5B%22CP00b0AP00b0AEsACCBGDeCgAAAAAAFAAAIwAAAQaQD2F2K2kKFgPi2QWYAQBCij4EAhQAAAAkCBIAAgAUgQAgFIIAwAIFAAAAAAAAAQEgCQAAQABAAAIACgAAAAAAIAAAAAAAQQAAAAAIAAAAAAAAEAQCAAAAQAAAAIAABEhCAAQQAEAAAAAAAQAAAAAAAAAAABAAAAAAAAAAAAAAAAAgAAgAA.YAAAAAAAAAA%22%2C%221~%22%2C%22905C9CAE-3D41-4084-95DA-144E5455EA05%22%5D%2Cnull%2Cnull%2C%5B%5D%5D"
```

Comment:
Are you sure these cookies will work in the long term? They contain IDs. Please either add a click rule or remove the rule if that doesn't work.